niceday-api/index.js

```diff
@@ -31,11 +37,37 @@ const options = {
 const expressAppConfig = oas3Tools.expressAppConfig(path.join(__dirname, 'api/openapi.yaml'), options);
 const app = expressAppConfig.getApp();
 
+function setupChat(therapistId, token) {
```

Comment:
Right... so, all this setup only needs to be done once (per session/token?). Before was it doing all this authentication stuff every time it sent a message?

Reply:
Yes, isn't this great?